### PR TITLE
Add common http client config

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose
Add common http client config

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Review by SkNuwanTissera — APPROVED** [review]

**Comment by SkNuwanTissera** [comment]
LGTM

File: .github/workflows/dev-stg-release.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+              if: github.event.inputs.bal_central_environment == 'DEV'
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args: push
+              env:
+                  BALLERINA_CENTRAL_ACCESS_TOKEN: ${{ secrets.BALLERINA_CENTRAL_DEV_ACCESS_TOKEN }}
+                  
+            - name: Push to Staging
+              if: github.event.inputs.bal_central_environment == 'STAGE'
+              uses: ballerina-platform/ballerina-action/@master
```
Comment (LakshanSS) [inline .github/workflows/dev-stg-release.yml:36]: Let's add the correct version here instead of `master`

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@master
+              with:
+                  args: pack
+            - name: Ballerina Push
+              uses: ballerina-platform/ballerina-action/@master
```
Comment (LakshanSS) [inline .github/workflows/release.yml:17]: Let's add the correct version

**Review by LakshanSS — APPROVED** [review]

**Comment by LakshanSS** [comment]
@SkNuwanTissera Could you send a PR to add the ballerina version as `2201.2.0` in the dev-staging release and prod release workflows?

**Pull request merged**